src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/HostedMcpServerToolCallContent.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>
+/// Represents a tool call request to a hosted MCP server.
+/// </summary>
+public class HostedMcpServerToolCallContent : AIContent
```

Comment:
You need to augment AIContent's polymorphism attributes, too, right?

Reply:
Yes, missed that.